### PR TITLE
fixes for mac casing

### PR DESCRIPTION
**Pull request by pgregorynz**
This was never merged back to the repo. Required to fix the mac casing issues

### CODE REVIEW

File: src/Bento.Website/Properties/launchSettings.json [hunk 1]
```diff
@@ -18,12 +18,12 @@
...
     "Umbraco.Web.UI": {
       "commandName": "Project",
-      "dotnetRunMessages": true,
       "launchBrowser": true,
       "applicationUrl": "https://localhost:44360;http://localhost:40560",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }      
+      },
+      "dotnetRunMessages": true
```
Comment (chrden) [inline src/Bento.Website/Properties/launchSettings.json]: Should this have been committed?

File: src/Bento.Website/Bento.Website.csproj [hunk 2]
```diff
@@ -30,4 +30,5 @@
 		<UserSecretsId>f423ebd9-7632-45cf-96cb-63b93dfda47c</UserSecretsId>
 	</PropertyGroup>
 
+	<PropertyGroup Condition=" '$(RunConfiguration)' == 'Umbraco.Web.UI' " />
```
Comment (chrden) [inline src/Bento.Website/Bento.Website.csproj]: Is this necessary for this change?

Comment (pgregorynz) [reply src/Bento.Website/Bento.Website.csproj]: Weird not sure where this came from but can remove

File: src/Bento.Website/Properties/launchSettings.json (hunk 1 above)
Comment (pgregorynz) [reply src/Bento.Website/Properties/launchSettings.json]: This looks like it was just moved?  will double check the file

**Review by chrden — APPROVED** [review]

**Pull request merged**